lib/quby/questionnaires/entities/flag.rb

```diff
@@ -18,6 +18,16 @@ def initialize(key:,
         def if_triggered_by(answer_flags)
           yield if answer_flags[key] == trigger_on
         end
+
+        def to_codebook(_options = {})
+          output = []
+          output << "#{key} flag"
+          output << " 'true' - #{description_true}"
+          output << " 'false' - #{description_false}"
+          output << " '' (leeg) - Vlag niet ingesteld, informatie onbekend"
```

Comment:
Zet dan een `(default)` achter 1 van de waardes. Want het niet zetten van een vlag is niet echt een edge-case ofzo, en dan wil je dus wel weten hoe die wordt geïnterpreteerd.